sdks/java/core/src/test/java/org/apache/beam/sdk/io/CountingSourceTest.java

```diff
@@ -79,6 +79,26 @@ public static void addCountingAsserts(PCollection<Long> input, long numElements)
       .isEqualTo(numElements - 1);
   }
 
+  public static void addCountingAsserts(PCollection<Long> input, long start, long end) {
```

Comment:
Why this duplication of tests: what's the difference between CountingSourceTest and CountingInputTest?